app/views/admin/_admin.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= javascript_include_tag 'admin'%>
+<%= stylesheet_link_tag    'admin', media: 'all' %>
+
+<%= render partial: 'navbar' %>]
+
+<% yield %>
```

Comment:
You should create different layout for admin and application, so you don't need this partial anymore.